src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -324,6 +327,8 @@
     text: az aks create -g MyResourceGroup -n MyManagedCluster --load-balancer-outbound-ip-prefixes <ip-prefix-resource-id-1,ip-prefix-resource-id-2>
   - name: Create a kubernetes cluster with basic SKU load balancer and AvailabilitySet vm set type.
     text: az aks create -g MyResourceGroup -n MyManagedCluster --load-balancer-sku basic --vm-set-type AvailabilitySet
+  - name: Create a kubernetes cluster with authorized apiserver ip ranges.
```

Comment:
nit: upper case "IP" instead of lower case "ip"

Reply:
Done